src/aiy/audio.py

```diff
@@ -69,3 +70,15 @@ def play_audio(audio_data):
 def say(words, lang='en-US'):
     """Says the given words in the given language with Google TTS engine."""
     aiy._drivers._tts.say(aiy.audio.get_player(), words, lang=lang)
+
+
+def get_status_ui():
```

Comment:
Not really... my local branch does not have it but when I fetch it came back... I guess I just have yet got the hang of git...